packages/extension/tests/utils.test.ts

```diff
@@ -69,6 +69,32 @@ test('extension manager removes native icon', () => {
   })
 })
 
+test('extension manager removes workspace modes', async () => {
+  const context = {
+    globalState: {
+      get: jest.fn().mockReturnValue({}),
+      setKeysForSync: jest.fn()
+    }
+  }
+  const extExport = activateGUI(context as any, {} as any)
+  // @ts-expect-error
+  vscode.workspace.workspaceFolders = ['/foo/bar']
+  ;(vscode.workspace.fs.readFile as jest.Mock).mockResolvedValue(JSON.stringify({
```

Comment:
Yeah, otherwise the compiler thinks you are doing this:

```js
vscode.workspace.workspaceFolders = ['/foo/bar'](vscode.workspace.fs.readFile as jest.Mock)...
```

That is one of the disadvantages of not having `;` everywhere but a small one I can live with. It is also weird to have to cast mocks everywhere in Jest. Vitest allows to do this `vi.mocked(...)`